README.md

```diff
@@ -13,3 +12,10 @@ Copy **session ID** and try below.
 ```
 ~$ ./index.js ceremony <session id>
 ```
+
+You can also use docker.
+
+```
+~$ docker run rootwarp/czg-keramony:latest index.js auth
+~$ docker run rootwarp/czg-keramony:latest index.js <session id>
+```
```

Comment:
Sorry, There is a typo on the repo name.
Please fix the name of the docker image.
`czg-keramony` -> `czg-keremony`